examples/examples/setup_client_custom_config.rs

```diff
@@ -0,0 +1,29 @@
+use subxt::{
+    config::{substrate::SubstrateExtrinsicParams, Config, SubstrateConfig},
+    OnlineClient,
+};
+
+/// Defing a custom config type:
+enum MyConfig {}
+impl Config for MyConfig {
+    // This is different from the default `u32`:
+    type Index = u64;
+    // We can point to the default types if we don't need to change things:
+    type Hash = <SubstrateConfig as Config>::Hash;
```

Comment:
Should we include a single code comment above each type with a 1-line explanation of what they are?

Reply:
For the sake of keeping such things in one place and uptodate, I think I'd in general lean towards people looking it up in the rust docs if interested (https://docs.rs/subxt/latest/subxt/config/trait.Config.html). We could def link to them though here :)

Reply:
I think a link would accomplish the same thing, as in pointing folks to the resource. Works for me!